README.md

```diff
@@ -135,6 +135,17 @@ sslCipherSuite | The prioritized list of allowed SSL/TLS ciphers. For more infor
 
 Similar to configuring the server, the client configuration file is located at `/etc/opt/omi/conf/omicli.conf`.
 
+### Collecting Performance Metrics using OMI/OMI Provider on docker container or virtual machine
+
+OMI/OMI provider can work on containers and machines, just make sure you know following notes:
+- OMI/OMI provider can collect virtual machine perf metrics even in the normal container. To do this, there are 3 ways docker provided: pseudo-files, docker stats command and Remote API, and it doesn't need to run with a privileged container, . 
```

Comment:
Change this to:

- OMI/OMI provider can collect virtual machine performance metrics even in a normal container. To do this, there are 3 ways docker supports this: pseudo-files, docker stats command and the Remote API. Docker doesn't need to run with a privileged container,